src/ctap2/credential_management.rs

```diff
@@ -110,5 +106,5 @@ pub struct Response {
     pub cred_protect: Option<CredentialProtectionPolicy>,
     // 0x0B
     #[serde(skip_serializing_if = "Option::is_none")]
-    pub large_blob_key: Option<Bytes<32>>,
```

Comment:
`serde(with = "serde_bytes")` works for `Option<[u8; 32]>`, `Option<&[u8; 32]>` and  `Option<&[u8]>`.

Reply:
But `serde_indexed` does not support `with`, see:
- https://github.com/trussed-dev/serde-indexed/issues/13
- https://github.com/trussed-dev/ctap-types/issues/40